snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/SnomedRf2ExportRequest.java

```diff
@@ -226,6 +257,39 @@ public Attachment execute(final BranchContext context) {
 		// Step 1: check if the export reference branch is a working branch path descendant
 		final TerminologyResource referenceCodeSystem = context.service(TerminologyResource.class);
 
+		if (referenceCodeSystem.getSettings() != null && !referenceCodeSystem.getSettings().isEmpty()) {
+			if (Strings.isNullOrEmpty(countryNamespaceElement)) {
+
+				if (maintainerType == null) {
+
+					String maintainerType = (String) referenceCodeSystem.getSettings().get("maintainerType");
```

Comment:
These config keys should be moved next to the currently available config keys. See `SnomedTerminologyComponentConstants`.